### PR TITLE
Mira oss move

### PR DESCRIPTION
**Pull request by abuokoene**
- Added the chart to the repo in the 'charts' folder
- Modified the circleci job to package the chart and push it to artifactory and bintray, only when the chart itself has been updated (i.e. anything under the charts/mira folder

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -349,6 +349,98 @@ jobs:
...
+          password: $DOCKER_PASSWORD
+    steps:
+      - attach_workspace:
+          at: ~/
+      - checkout
+      - run:
+          name: Push
+          command: |
+            #Check if the chart has been updated
+            result=$(python charts/check_git_py.py -c charts/mira)
```
Comment (FredrikFolkesson) [inline .circleci/config.yml]: Can we check that the chart _version_ has been updated? I am thinking we want to make sure that the pushed chart is not the same version as the old one (someone fixing something small and forgets to update the version could easily happen)

File: .circleci/config.yml [hunk 2]
```diff
@@ -349,6 +349,98 @@ jobs:
...
+              #Get the filename of the packaged chart
+              new_tgz="$(ls mira*.tgz)"
+
+              #Get Jfrog cli
+              curl -Lo jfrog https://api.bintray.com/content/jfrog/jfrog-cli-go/\$latest/jfrog-cli-linux-386/jfrog?bt_package=jfrog-cli-linux-386
+              chmod a+x ./jfrog
+
+              #Deploy chart to artifactory
+              echo "Uploading chart to artifactory"
+              ./jfrog rt u $new_tgz charts-oss --url https://qliktech.jfrog.io/qliktech --apikey $RT_APIKEY
```
Comment (FredrikFolkesson) [inline .circleci/config.yml]: Why are we pushing to both Artifactory and Bintray? :)

Comment (abuokoene) [reply .circleci/config.yml]: Artifactory is only accessible internally, so pushing the chart to Bintray makes it public

File: .circleci/config.yml (hunk 1 above)
Comment (abuokoene) [reply .circleci/config.yml]: Yeah, that makes sense. Will push a change for that

File: .circleci/config.yml (hunk 2 above)
Comment (FredrikFolkesson) [reply .circleci/config.yml]: why not only Bintray then?

Comment (abuokoene) [reply .circleci/config.yml]: The qsefe chart looks at artifactory for getting its subcharts when being packaged

Comment (FredrikFolkesson) [reply .circleci/config.yml]: Ah okay.

**Pull request merged**